src/commands/compress.rs

```diff
@@ -99,6 +105,8 @@ pub fn compress_files(
                     .compression_level(gzp::par::compress::Compression::new(
                         level.map_or_else(Default::default, |l| (l as u32).clamp(0, 9)),
                     ))
+                    .num_threads(logical_thread_count())
+                    .expect("gpz: num_threads must be greater than 0")
```

Comment:
`num_threads` will err if the input is `0`, but works otherwise. This should never happen due to `parse_and_validate_args`

Also, using `logical_thread_count` makes sense here since `gzp` defaults to `num_cpu::get()`